packages/dds/tree/src/feature-libraries/forest-summary/forestSummarizer.ts

```diff
@@ -35,6 +35,8 @@ import { ICodecOptions, noopValidator } from "../../codec/index.js";
 import { FieldBatchEncodingContext, FieldBatchCodec } from "../chunked-forest/index.js";
 // eslint-disable-next-line import/no-internal-modules
 import { chunkField, defaultChunkPolicy } from "../chunked-forest/chunkTree.js";
+// eslint-disable-next-line import/no-internal-modules
+import { DetachedNodeBuild } from "../../core/tree/delta.js";
```

Comment:
For this import we should probably follow the pattern established by the import of DeltaFieldChanges: import DeltaDetachedNodeBuild from "../../core/index.js";

Reply:
Updated import to `DeltaDetachedNodeDestruction`